### PR TITLE
feat(card): Added kd-card component from shidoka application

### PR DESCRIPTION
**Pull request by maulik-thaker1**
## Summary

- Added kd-card component from shidoka-application repo with simple content inside

## Checklist

- [x] Used Conventional Commit messages as outlined in the contributing guide.
  - [x] Noted breaking changes (if any).
- [x] Documented/updated all props, events, slots, parts, etc with JSDoc.
  - [x] Ran the `analyze` command to update Storybook docs.
- [x] Added/updated Stories with controls to cover all variants.
- [x] Added/updated a basic unit test, at least, for each component.
- [x] Added/updated the Figma link for the Story's Design tab.

### CODE REVIEW

File: src/components/card/card.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  @property({ type: String })
+  rel = '';
+
+  /** Defines a target attribute for where to load the URL in case of clickable card. Possible options include "_self" (deafult), "_blank", "_parent", "_top" */
+  @property({ type: String })
+  target = '_self';
+
+  /** Hide card border. Useful when clickable card use inside `<kyn-notification>` component. */
+  @property({ type: Boolean })
+  hideBorder = false;
```
Comment (brian-patrick-3) [inline src/components/card/card.ts:35]: Maybe we should apply this to the "normal" variant as well, otherwise it's kind of confusing that it does nothing.

**Review by brian-patrick-3 — APPROVED** [review]

File: src/components/card/card.ts (hunk 1 above)
Comment (maulik-thaker1) [reply src/components/card/card.ts:35]: yes normal variant is there. Story name `Simple`

**Comment by maulik-thaker1** [comment]
so much lint error in checks while raising this PR. Its related to accordion and button. Although I didn't change anything inside this component

**Pull request merged**